docs/excel/custom-functions-authentication.md

```diff
@@ -1,51 +1,51 @@
 ---
-ms.date: 04/15/2019
+ms.date: 04/23/2019
 description: Authenticate users using custom functions in Excel.
 title: Authentication for Custom Functions
 ---
 
 # Authentication
 
-In some scenarios your custom function will need to authenticate the user in order to access protected resources. While custom functions don't require a specific method of authentication, you should be aware that custom functions run in a separate runtime from the task pane and other UI elements of your add-in. Because of this, you'll need to pass data back and forth between the two runtimes using the `AsyncStorage` object and the Dialog API.
+In some scenarios your custom function will need to authenticate the user in order to access protected resources. While custom functions don't require a specific method of authentication, you should be aware that custom functions run in a separate runtime from the task pane and other UI elements of your add-in. Because of this, you'll need to pass data back and forth between the two runtimes using the `Office.Storage` object and the Dialog API.
   
-## AsyncStorage object
+## Office.Storage object
 
-The custom functions runtime doesn't have a `localStorage` object available on the global window, where you might typically store data. Instead, you should share data between custom functions and task panes by using [OfficeRuntime.AsyncStorage](/javascript/api/office-runtime/officeruntime.asyncstorage) to set and get data.
+The custom functions runtime doesn't have a `localStorage` object available on the global window, where you might typically store data. Instead, you should share data between custom functions and task panes by using [Office.Storage](/javascript/api/office-runtime/officeruntime.asyncstorage) to set and get data.
 
-Additionally, there is a benefit to using `AsyncStorage`; it uses a secure sandbox environment so that your data cannot be accessed by other add-ins.
+Additionally, there is a benefit to using `Office.Storage`; it uses a secure sandbox environment so that your data cannot be accessed by other add-ins.
```

Comment:
```suggestion
Additionally, there is a benefit to using the `Storage` object; it uses a secure sandbox environment so that your data cannot be accessed by other add-ins.
```
from here on I would just use "Storage object"